package com.gla.Inheritance;

    class Bird extends Animal {

        public Bird(String name, int age) {
            super(name, age);

        }

        @Override
        public void makeSound() {
            System.out.println("Bird chirps: Tweet Tweet");
        }
    }

